### ISSUE TITLE
Custom label on a device with a boolean value

### ISSUE THREAD

**Issue by daneedk** [athombv/apps-sdk-issues#21]
In V1 it was possible to configure a custom label for a boolean value on a device. At this moment in V2 the standard values are `Yes` or `No`. For a lot of circumstances other descriptions would make more sense, for example `Active`/`Idle` or `Open`/`Closed` etc. 
Ted Tolboom asked about this on slack [here](https://athomcommunity.slack.com/archives/C04SUGZ9E/p1545814617136100) and @weejewel responded with `That's not implemented (yet - to be discussed if we want to keep supporting this)`
I hope the result will be that you keep supporting this to give end users the best experience possible.

**Comment by TedTolboom** [comment]
I think this is a must have if the goal is to provide the best user experience. 

Users are used to this in the v1 app / environment, so showing only yes or no feels medieval to me.

**Comment by robertklep** [comment]
I agree as well: it may break the UX for existing apps, and "Yes/No" doesn't always convey the proper meaning behind the capability.

As @daneedk states, for certain devices, "Open/Closed" makes much more sense, or "On/Off" (for instance, I'm using "On/Off" for a boolean "Verwarming").

**Comment by WeeJeWel** [comment]
You can start adding `titleTrue` and `titleFalse` as capability option, we'll make sure it gets displayed.

**Comment by WeeJeWel** [comment]
Example:

```
"capabilitiesOptions": {
 "my_sensor": {
    "titleTrue": {
       "en": "True Value"
    },
    "titleFalse": {
       "en": "False Value"
    }
  }
}
```

**Comment by daneedk** [comment]
Thank you.

**Comment by TedTolboom** [comment]
Thanks.
Updated 3 of my apps with it; waiting for app store approval ;-P

**Comment by WeeJeWel** [comment]
Added in Homey v2.0.1.

**Comment by Christian-Athom** [comment]
This will be available in the Mobile App version 2.0.3.247
